test/sys/test_ptrace.rs

```diff
@@ -115,6 +115,49 @@ fn test_ptrace_cont() {
     }
 }
 
+// ptrace::interrupt is only available in these platforms
+#[cfg(all(target_os = "linux", not(any(target_arch = "mips", target_arch = "mips64"))))]
```

Comment:
Really?  libc defines it for all architectures.

Reply:
See https://github.com/nix-rust/nix/blob/68d01f0426a82b485b22c31425c0e53b5c372a7d/src/sys/ptrace/linux.rs#L105

Reply:
Yes I know.  But you added that line.  So why did you leave out mips?

Reply:
I mean, my rationale for adding the line (at `test/sys/test_ptrace.rs` is because this pull request depends on `nix::Request::PTRACE_INTERRUPT` to be available. 

The test would fail without me adding that limitation (which is similar than for PTRACE_INTERRUPT). For why that has been added for Request::PTRACE_INTERRUPT initially, I have no idea.

Reply:
My apologies.  I saw that you added `fn interrupt`, and didn't realize that `PTRACE_INTERRUPT` already existed.  But none the less, I can see no reason to restrict it on mips.  Could you please remove the restriction?